File: abies/config/strings.py
"""App-wide user-facing string constants.

Locale-specific text (UI labels, button copy, error messages, column
headers) lives in `strings_<lang>.py` and is re-exported below — change
the import line to switch language.

For internal symbolic identifiers (JSON field names, API envelope keys,
status enum values, etc.), see `config/constants.py`.
"""

from config.strings_it import *  # noqa: F401,F403

File: abies/apps/base/static/base/js/strings.js
/**
 * App-wide string constants.
 *
 * Locale-specific strings live in `strings_<lang>.js` and are re-exported
 * below — change the import line to switch language.  Locale-INDEPENDENT
 * identifiers (wire markers, internal column names, etc.) are defined
 * directly in this file so they aren't duplicated across language files.
 */

export * from './strings_it.js';

// Internal digest column identifiers — wire format, never displayed.
export const COL_ROW_ID  = 'row_id';
export const COL_VERSION = 'version';
